src/screens/ShareScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import { StatusBar } from 'expo-status-bar';
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export default function ShareScreen() {
```

Comment:
```suggestion
export default function ImageDetailsScreen() {
```
It will be not only for sharing but to display/edit text content and other details